packages/craco-esbuild/src/index.js

```diff
@@ -12,15 +12,20 @@ module.exports = {
     context: { paths },
   }) => {
     const useTypeScript = fs.existsSync(paths.appTsConfig);
+    const esbuildLoaderOptions = pluginOptions.esbuildLoaderOptions;
```

Comment:
`pluginOptions` can be null if you don't pass options so this could break.
```suggestion
    const esbuildLoaderOptions = pluginOptions ? pluginOptions.esbuildLoaderOptions : {};
```

Reply:
Added suggestion